src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -123,15 +132,17 @@
     }
 
     /**
-     * Returns a {@code Set} of non-binary files for the repo {@code repoConfig}.
+     * Returns a {@code Set} of non-binary files for the repo {@code repoConfig}
+     * if {@code isBinaryFiles} is set to `false`.
+     * Otherwise, returns a {@code Set} of binary files for the repo {@code repoConfig}
      */
-    public static Set<Path> getNonBinaryFilesList(RepoConfiguration repoConfig) {
+    public static Set<Path> getFilesList(RepoConfiguration repoConfig, boolean isBinaryFile) {
```

Comment:
`List` can be omitted, since what its really returning is a Set anyway.

Reply:
FIxed.